modules/Core/module.php

```diff
@@ -478,6 +478,14 @@ public function __construct(Language $language, Pages $pages, User $user, Naviga
             ]
         );
 
+        EventHandler::registerEvent('userNewProfilePost',
+            $language->get('admin', 'user_new_profile_post_hook_info')
+        );
+
+        EventHandler::registerEvent('userProfilePostReply',
+            $language->get('admin', 'user_profile_post_reply_hook_info')
+        );
+
```

Comment:
You are missing the values that the event returns and their description. These should be 

- username
- content
- content_full
- avatar_url
- title
- url

Reply:
Thanks - I've updated the events to show the return values and descriptions now.